master/internal/task/allocation.go

```diff
@@ -170,6 +170,7 @@ func newAllocation(
 		req: req,
 		model: model.Allocation{
 			AllocationID: req.AllocationID,
+			// do we want to set the starttime?
```

Comment:
I think startTime is when an allocation starts running / pulling so no we don't want to set start time here?